### DISCUSSION TITLE
max_http_buffer_size modification

### DISCUSSION THREAD

**Discussion by Piguite** [zauberzeug/nicegui#4081] [Ideas / Feature Requests]
### Description

Hey, i wanted to know if there was an "official" way to modify the max_http_buffer_size from socketio because the default value is 1Mo. ATM I did this to change the default value and it work despite being a bit brutal : 
```
core.sio = sio = socketio.AsyncServer(async_mode='asgi', cors_allowed_origins='*', json=json, max_http_buffer_size = 10*1024*1024)
```
(nicegui/nicegui.py line 49)

**Comment by rodja** [comment]
We discussed something like this in https://github.com/zauberzeug/nicegui/issues/3410 but have not concluded how the configuration should happen.
